interface/src/app/gene/network/network.js

```diff
@@ -187,20 +205,22 @@ angular.module('adage.gene.network', [
           NodeService.get(
             {'heavy_genes': heavyGenes, 'limit': 0},
             function success(response) {
-              var n = response.objects.length;
+              var i = 0, n = response.objects.length;
+              var uri;
               str += '<br />' + n + (n > 1 ? ' nodes are ' : ' node is ');
               str += 'related to both genes' + (n > 0 ? ':' : '.');
-              for (var i = 0; i < n; ++i) {
-                str += '<br />* ' + response.objects[i].name + '</li>';
+              for (; i < n; ++i) {
+                uri = '<a href="#/node/' + response.objects[i].id + '">';
+                str += '<br />* ' + uri + response.objects[i].name + '</a>';
               }
-
               edgeTip.html(str);
               edgeTip.show(data, target);
             },
-            function error(err) {
-              $log.error(
-                'Failed to get node info for gene edge: ' + err.statusText);
-              str += 'Can\'t get node information, please try again later.';
+            function error(response) {
+              var message = 'Failed to get node info for gene edge: ' +
+                  response.statusCode + ' ' + response.statusText;
+              $log.error(message);
+              str += message + '. Please try again later.';
```

Comment:
Did you test this error case? I suspect you will need a `<br>` (as you have at line 214 above) in order to have the whole `str` formatted properly.